controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -51,6 +51,7 @@ const (
 	unmanagedRetryDelay           = time.Minute * 10
 	provisionerNotReadyRetryDelay = time.Second * 30
 	rebootAnnotationPrefix        = "reboot.metal3.io"
+	inspectAnnotation             = "inspect.metal3.io"
```

Comment:
@hardys since you are also introducing the inspect annotation, I will rebase once https://github.com/metal3-io/baremetal-operator/pull/787 is merged.

Reply:
I have rebased my changes based on #787